src/lavinmq/queue/message_store.cr

```diff
@@ -282,11 +283,13 @@ module LavinMQ
             end
             @replicator.try &.register_file(file)
           end
+          log_progress_and_yield("Loading acks...") if (count += 1) % 100 == 0
```

Comment:
I think it would be nice to include which file it just loaded, and maybe how include the count value as well? Same for all other `log_progress_and_yield` where applicable.